inspirehep/modules/forms/static/js/forms/affiliations_typeahead.js

```diff
@@ -80,14 +80,14 @@ define([
         }.bind(this));
       }.bind(this),
       displayKey: function(data) {
-        return data.metadata.ICN[0];
+        return data.metadata.legacy_ICN;
       },
       templates: {
         empty: function(data) {
           return 'Cannot find this affiliation in our database.';
         },
         suggestion: function(data) {
-          if (data.new_ICN != data.ICN) {
+          if (data.ICN != data.legacy_ICN) {
```

Comment:
I don't know if it needs to be an array, we didn't really work on institutions. Probably not, but for now it is (might change in the future).